test/Microsoft.AspNetCore.Razor.Test.Common/Language/CodeGeneration/TestCodeRenderingContext.cs

```diff
@@ -15,7 +15,7 @@ public static class TestCodeRenderingContext
         {
             var codeWriter = new CodeWriter();
             var documentNode = new DocumentIntermediateNode();
-            var options = RazorCodeGenerationOptions.CreateDesignTimeDefault();
+            var options = RazorCodeGenerationOptions.CreateDesignTime(c => { });
```

Comment:
Ah, this is why you wanted a `CreateDesignTimeDefault`. It's a little ugly for tests but I think it's worth the lesser public API (people other than us will never need to do this).

As a side note, for lambdas where you don't use the variable it's common practice to use the `_` character